src/App.js

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
-// import logo from './logo.svg';
-// import './App.css';
-import Header from './components/header';
+import Header from './components/Header';
```

Comment:
Hey Steve! It looks like the file name didn't get pushed up and capitalized so this is causing issues when you try and run npm start.